doc/man/scons.xml

```diff
@@ -6940,37 +6945,40 @@ you can leave off the target file suffix,
 and &scons; will add it automatically.</para>
 
 <programlisting language="python">
-bld = Builder(action = 'pdftex &lt; $SOURCES &gt; $TARGET'
-              suffix = '.pdf',
-              src_suffix = '.tex')
-env = Environment(BUILDERS = {'PDFBuilder' : bld})
-env.PDFBuilder(target = 'foo.pdf', source = 'foo.tex')
+bld = Builder(
+    action='pdftex &lt; $SOURCES &gt; $TARGET',
+    suffix='.pdf',
+    src_suffix='.tex'
+)
+env = Environment(BUILDERS={'PDFBuilder': bld})
+env.PDFBuilder(target='foo.pdf', source='foo.tex')
 
 # The following creates "bar.pdf" from "bar.tex"
-env.PDFBuilder(target = 'bar', source = 'bar')
+env.PDFBuilder(target='bar', source='bar')
 </programlisting>
 
-<para>Note also that the above initialization
-overwrites the default Builder objects,
-so the Environment created above
-can not be used call Builders like
-<methodname>env.Program</methodname>,
-<methodname>env.Object</methodname>,
-<methodname>env.StaticLibrary</methodname> etc.</para>
+<para>Note that the above initialization
+replaces the default dictionary of Builders,
+so this &consenv; can not be used call Builders like
```

Comment:
Should this be "can not be used to call Builders..."

Reply:
I seem to be blind, isn't that what it says?

Reply:
Nope.. it's missing "used **to** call Builders..."

Reply:
I'll merge and fix.

Reply:
Ahh. misread... xml blinded.. ;)

Reply:
no, i missed the "to"